packages/landing/static/index.css

```diff
@@ -169,6 +169,8 @@ button {
 
 .debug__value {
   display: flex;
+  overflow: hidden;
+  white-space:nowrap;
```

Comment:
nit: should be a ` ` between the : and the word, odd do we not have a linter/formatter for this?

Reply:
Good pick.